tests/test_mapper.py

```diff
@@ -394,7 +394,7 @@ def test_mapper_lingling5(self):
         # 'realistic' circuit
         v = 'lingling5'
         config = "cc_light.s17"
-        num_qubits = 7
+        num_qubits = 17
```

Comment:
Has the meaning of 'num_qubits' changed from virtual number to physical number?

Reply:
It's because while routing on s17, higher qubit indices were reached.
num_qubits is the # of physical qubits, as discussed